addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -354,3 +354,6 @@ Many2ManyTagsFieldColorEditable.extractProps = (params) => {
 };
 
 registry.category("fields").add("form.many2many_tags", Many2ManyTagsFieldColorEditable);
+
+registry.category("fields").add("calendar.one2many", Many2ManyTagsField);
+registry.category("fields").add("calendar.many2many", Many2ManyTagsField);
```

Comment:
is this necessary? I wasn't aware the calendar view looks in the field registry.